app/services/sign_in/user_creator.rb

```diff
@@ -28,6 +28,8 @@ def initialize(user_attributes:, state_payload:, verified_icn:, request_ip:)
       @mhv_correlation_id = user_attributes[:mhv_correlation_id]
       @verified_icn = verified_icn
       @request_ip = request_ip
+      @first_name = user_attributes[:first_name]
```

Comment:
Can you add these to the `attr_reader` above and then refer to these without the `@` symbol below?